src/engraving/compat/readscorehook.cpp

```diff
@@ -22,7 +22,8 @@
 #include "readscorehook.h"
 #include "readstyle.h"
 
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
+#include "libmscore/masterscore.h"
```

Comment:
Twice the same #include here??

Reply:
Oops )
I added `#include "libmscore/masterscore.h"` first
And after I decided to just change `#include "libmscore/score.h"` to `#include "libmscore/masterscore.h"`

Reply:
Fixed